Dockerfile

```diff
@@ -62,4 +63,4 @@ ENTRYPOINT ["/ckan-entrypoint.sh"]
 USER ckan
 EXPOSE 5000
 
-CMD ["ckan-paster","serve","/etc/ckan/production.ini"]
+CMD ["ckan","-c","/etc/ckan/production.ini", "run", "--host", "0.0.0.0"]
```

Comment:
Had to set the host to `0.0.0.0` to allow host network to access the server in the container. The `ckan run` command defaults to `localhost` if `--host` is not provided.